packages/twenty-front/src/modules/ui/input/components/TextInputV2.tsx

```diff
@@ -143,7 +144,7 @@ const TextInputV2Component = (
       {label && <StyledLabel>{label + (required ? '*' : '')}</StyledLabel>}
       <StyledInputContainer>
         <StyledInput
-          autoComplete="off"
+          autoComplete={autoComplete || 'off'}
```

Comment:
Seeking to disable autocomplete here as it does not make sense for the db and will overwrite the default values from data.
"off" was not working (maybe it's not doing what I think it tries to do - as I don't see why we would want to always disable autocomplete?).

https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion

(...)_For this reason, many modern browsers do not support autocomplete="off" for login fields:

If a site sets autocomplete="off" for a [<form>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form), and the form includes username and password input fields, then the browser still offers to remember this login, and if the user agrees, the browser will autofill those fields the next time the user visits the page.
If a site sets autocomplete="off" for username and password [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) fields, then the browser still offers to remember this login, and if the user agrees, the browser will autofill those fields the next time the user visits the page._